packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -87,4 +97,28 @@ export class TableInputCSV extends React.PureComponent<Props, State> {
   }
 }
 
-export default TableInputCSV;
+export const TableInputCSV = withTheme(unThemedTableInputCSV);
+TableInputCSV.displayName = 'TableInputCSV';
+
+const getStyles = stylesFactory(
+  (theme: GrafanaTheme): TableInputCSVStyle => {
+    return {
+      TableInputCsv: css`
+        position: relative;
+        textarea {
```

Comment:
Nit: It could be a nice addition to swap these element selectors out for class names if possible. My reasoning is that element selectors tie semantics to styling which can lead to increased maintenance as a future developer who edits the HTML tags will also need to change the CSS selectors. Class names allows us to separate these concerns.

Reply:
Alright 👍
Thanks @jackw